public-images/public-images-reference/artifacts.rst

```diff
@@ -0,0 +1,1501 @@
+
+       The VHD files we publish are tailored specifically for use in the Azure cloud. They do not contain standard VM images and will not function outside of Azure, including on-premises Hyper-V or local development environments. 
+
+Other files
+-----------
+This section includes information on checksums, GPG signatures, changelogs and manifest files. These files help verify the integrity and authenticity of the images, provide details on changes between versions and list all included packages.
+
+Changelogs
+~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``.image_changelog.json``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64.daily.20240612.20240622.image_changelog.json``
+   * - Format description
+     - JSON-formatted changelogs detailing the changes in the respective image builds.
+   * - Data format
+     -
+      | - **summary:**
+      |   - **snap:**
+      |     - **added:** Lists newly added snap packages.
+      |     - **removed:** Lists removed snap packages.
+      |     - **diff:** Lists snap packages that have changed.
+      |   - **deb:**
+      |     - **added:** Lists newly added deb packages.
+      |     - **removed:** Lists removed deb packages.
+      |     - **diff:** Lists deb packages that have changed.
+      | - **diff:**
+      |   - **deb:** Detailed information on changed deb packages:
+      |     - **name:** The name of the package.
+      |     - **from_version:** Previous version details, including the source package name and version.
+      |     - **to_version:** New version details, including the source package name and version.
+      |     - **cves:** Common Vulnerabilities and Exposures fixed.
+      |     - **launchpad_bugs_fixed:** IDs of fixed Launchpad bugs.
+      |     - **changes:** A list of changes with details like CVEs, change logs, package name, version,
+      |       urgency, distributions, author and date.
+      |     - **notes:** Additional notes, if any.
+      |   - **snap:** Detailed information on changed snap packages (same structure as deb).
+      | - **added:** Lists newly added deb and snap packages.
+      | - **removed:** Lists removed deb and snap packages.
+      | - **notes:** General notes regarding the changelog.
+      | - **from_series:** The series name of the previous image (e.g. `noble`). 
+      | - **to_series:** The series name of the current image.
+      | - **from_serial:** The serial number of the previous image (e.g. `20240612`).
+      | - **to_serial:** The serial number of the current image.
+      | - **from_manifest_filename:** Filename of the previous manifest.
+      | - **to_manifest_filename:** Filename of the current manifest.
+   * - Example
```

Comment:
This example is quite long. It would be nice to trim it to still be realistic, but only provide just enough examples to get the point across.

Reply:
Agreed. Having to add the collapsible element should have been enough of a smell to shorten it.  I'll look at modifying it to be more concise.

Reply:
Shortened to 1 example for each of diff, added, and removed. Considerably more readable.